CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Clarify where and how dotted keys define tables.
 * Add new `\e` shorthand for the escape character.
+* Seconds in Date-Time and Time values are now optional.
```

Comment:
```suggestion
* Seconds in `Date-Time` and `Time` values are now optional.
```

Not sure what's the rule in formatting here, after all it's just a changelog, just thought this might be better ;).

Reply:
It's good as it is. If you look at v0.5.0, they spell Date and Date-Time unformatted, the same way I did it in the PR.